packages/cedar/src/dataset.ts

```diff
@@ -1,16 +1,17 @@
-import { IFeature, IFeatureSet } from '@esri/arcgis-rest-common-types'
+import { IField as IRestField, IFeature, IFeatureSet } from '@esri/arcgis-rest-common-types'
 
 export interface IField {
```

Comment:
TODO: deprecate this?

Reply:
unfortunately not. your `IField` interface is its own beast.